src/CommonLib/hostfxr_utility.cpp

```diff
@@ -204,8 +221,29 @@ HOSTFXR_UTILITY::GetHostFxrParameters(
     }
     else
     {
-        if (FAILED(hr = HOSTFXR_UTILITY::FindDotnetExePath(&struExeLocation)))
+        if (fIsDotnetPath)
         {
+            // If dotnet was specified in the processPath, find dotnet. 
+            if (FAILED(hr = HOSTFXR_UTILITY::FindDotnetExePath(&struExeLocation)))
+            {
+                goto Finished;
+            }
+        }
+        else
+        {
+            // If the processPath file does not exist and it doesn't include dotnet.exe or dotnet
+            // then it is an invalid argument.
+            hr = E_INVALIDARG;
+            if (SUCCEEDED(struEventMsg.SafeSnwprintf(
+                L"Invalid or unknown processPath provided in web.config: processPath = %s, ErrorCode = '0x%x'.",
```

Comment:
for now, don't hardcode the string here as it will block localization in the future.

Reply:
Do these all still go in resources.h?

Reply:
yes